### PR TITLE
rename function variables

### PR DESCRIPTION
**Pull request by bc118**
- Clean up some of the `fit_dihedral_with_gomc` naming functions.  Change the names for easier user interpretation.

- removed some duplicate files.

### CODE REVIEW

File: docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst [hunk 1]
```diff
@@ -72,27 +73,52 @@ Run the dihedral fit to fit to the MM simulations:
...
+
 
 Fit a dihedral for propionic acid using Gaussian-style output files
 -------------------------------------------------------------------
 The propionic acid CT-CT-C-OH dihedral fit example is provided below. This is an example where the CT-CT-C-OH is fit for both
 CT-CT-C-OH and CT-CT-C-O, by fitting the CT-CT-C-OH and zeroing the CT-CT-C-O, which makes it easier to fit.
 
 Additionally, one can perform the dihedral fit with QM calculations with any QM engine as long as they are in the
-`Gaussian style output files format <https://github.com/GOMC-WSU/MoSDeF-dihedral-fit/tree/main/mosdef_dihedral_fit/utils/files/gaussian_style_output_files>`_
+`Gaussian style output files format <https://github.com/bc118/MoSDeF-dihedral-fit/tree/main/mosdef_dihedral_fit/tests/files/gaussian_style_output_files>`_
```
Comment (CalCraven) [inline docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst:121]: Do you want this to point to your branch of this package and not the parent one at GOMC-WSU?

File: mosdef_dihedral_fit/dihedral_fit/fit_dihedral_with_gomc.py [hunk 2]
```diff
@@ -597,6 +607,19 @@ def fit_dihedral_with_gomc(
     else:
         raise TypeError(print_error_value)
 
+    # Check if 'combining_rule' is correct
+    # Set the VDWGeometricSigma variable from the combining_rule
+    if combining_rule == None:
```
Comment (CalCraven) [inline mosdef_dihedral_fit/dihedral_fit/fit_dihedral_with_gomc.py:612]: Yeah this is a nice enough way to merge the APIs for dihedral-fit and mosdef-gomc. Just means the top level arguments are going to much more clear and easy to find, and it's clear from the input what the arguments for mosdef-gomc would correspond to.

**Review by CalCraven — APPROVED** [review]
One minor comment, but otherwise it looks good.

File: mosdef_dihedral_fit/dihedral_fit/fit_dihedral_with_gomc.py (hunk 2 above)
Comment (bc118) [reply mosdef_dihedral_fit/dihedral_fit/fit_dihedral_with_gomc.py:612]: cool!

File: docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst (hunk 1 above)
Comment (bc118) [reply docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst]: Good Catch.  Changed now

**Pull request merged**